packages/android_alarm_manager/android/build.gradle

```diff
@@ -1,3 +1,16 @@
+def PLUGIN = "android_alarm_manager";
+def ANDROIDX_WARNING = "flutterPluginsAndroidXWarning";
+gradle.buildFinished { buildResult ->
+    if (buildResult.failure && !rootProject.ext.has(ANDROIDX_WARNING)) {
+        println '         *********************************************************'
+        println 'WARNING: This version of ' + PLUGIN + ' will break your Android build if it isn\'t compatible with AndroidX.'
+        println '         https://goo.gl/y9dCcU describes AndroidX and how to migrate to it using Android Studio.'
+        println '         https://goo.gl/n2E4mD describes how to open a Flutter app\'s Android files for refactoring in Android Studio.'
```

Comment:
Maybe just one more line saying that this error message prints out for any build failure, and the failure may be unrelated to this problem but that users should still validate it.  

Ideally we detect that the failure was for this problem and help them even more, but this is at least better than what we have.